### ISSUE TITLE
Change hit timing distribution shape

### ISSUE THREAD

**Issue by YujiOkajima** [WCSim/WCSim#53]
In WCSim, the hit timing distribution is based on Gaussian.
However, All photosensors timing distribution seems to be asymmetric and not to follow Gaussian.
The figure below is hit timing distribution of 1 p.e. charge of High QE R3600.

![pmt_about1pe](https://cloud.githubusercontent.com/assets/8589041/6391522/68416124-bdfa-11e4-979c-1fd28d63101b.png)

**Comment by cwwalter** [comment]
Interesting plot!  But, keep in mind that the time smearing in WCSim (now) is not the raw time distribution of the tube but rather the equivalent smearing after it comes out of the TQ map correction which corrects for time walk etc.  That time is the start time of the pulse.

So, the Gaussian smearing is how it is done in skdetsim.  The MCs don't simulate TQ map correction.  This needs further discussion if we wanted this fundamental approach.

**Comment by YujiOkajima** [comment]
I'm sorry for late response.

The plot I appended on the previous comment, I have already corrected the TQ map.
The figure below shows relationship between charge and the peak of timing distribution before correction
![tqcurve_nocorr](https://cloud.githubusercontent.com/assets/8589041/6438319/418a269c-c10b-11e4-8ae2-b277792a60d7.png)

And the figure below shows that after TQ map correction 
![tqcurve_corr](https://cloud.githubusercontent.com/assets/8589041/6438364/a5f1397c-c10b-11e4-9963-0c2822bd7e18.png)
From this figure, the correction seems to be successfully done.
